### ISSUE TITLE
Responsive in both directions

### ISSUE THREAD

**Issue by stellanhaglund** [hootsuite/grid#91]
Is there any way to make sure a certain amount of rows and columns such as 5 columns and 3 rows will always fit the container?

**Comment by NiGhTTraX** [comment]
No. The grid is designed to expand infinitely in one direction. It could probably be changed to allow for restrictions on both axes, but that's not something on our roadmap. PRs are welcome.

**Comment by RonakKhandelwal** [comment]
+1 @stellanhaglund found a work around yet?
